#! /bin/sh -e 
## button_label.dpatch by <luther@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Makefiles, configure scripts and other build stuff adapted for
## DP: debian package creation

if [ $# -ne 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0
@DPATCH@
diff -ur lablgtk2-2.2.0.orig/src/gButton.ml lablgtk2-2.2.0/src/gButton.ml
--- lablgtk2-2.2.0.orig/src/gButton.ml	2003-09-21 12:02:17.000000000 +0200
+++ lablgtk2-2.2.0/src/gButton.ml	2003-10-10 15:10:25.000000000 +0200
@@ -14,6 +14,8 @@
   method clicked () = Button.clicked obj
   method set_relief = set Button.P.relief obj
   method relief = get Button.P.relief obj
+  method set_label = set Button.P.label obj
+  method label = get Button.P.label obj
   method grab_default () =
     set Widget.P.can_default obj true;
     set Widget.P.has_default obj true
diff -ur lablgtk2-2.2.0.orig/src/gButton.mli lablgtk2-2.2.0/src/gButton.mli
--- lablgtk2-2.2.0.orig/src/gButton.mli	2003-09-27 15:42:19.000000000 +0200
+++ lablgtk2-2.2.0/src/gButton.mli	2003-10-10 15:10:42.000000000 +0200
@@ -17,6 +17,8 @@
     method clicked : unit -> unit
     method set_relief : Tags.relief_style -> unit
     method relief : Tags.relief_style
+    method set_label : string -> unit
+    method label : string
     method grab_default : unit -> unit
     method event : event_ops
   end
